src/cipher/crypto.go

```diff
@@ -71,20 +119,27 @@ func MustPubKeyFromHex(s string) PubKey {
 func PubKeyFromHex(s string) (PubKey, error) {
 	b, err := hex.DecodeString(s)
 	if err != nil {
-		return PubKey{}, errors.New("Invalid public key")
+		return PubKey{}, ErrInvalidPubKey
 	}
 	return NewPubKey(b), nil
 }
 
 // PubKeyFromSecKey recovers the public key for a secret key
 func PubKeyFromSecKey(seckey SecKey) PubKey {
 	if seckey == (SecKey{}) {
-		log.Panic("PubKeyFromSecKey, attempt to load null seckey, unsafe")
+		err := skyerrors.NewValueError(
```

Comment:
Don't linebreak args, there is no column limit

Reply:
I use vim inside a container . Long lines are clipped , so I try to stick to 80 chars column limit

Reply:
for instance  <img width="585" alt="screen shot 2018-08-07 at 7 25 49 pm" src="https://user-images.githubusercontent.com/538253/43807722-c9e29404-9a77-11e8-86f2-ccd854d659b5.png">